src/CMakeLists.txt

```diff
@@ -453,6 +453,10 @@ if(ALICEVISION_BUILD_SFM)
   endif()
 endif()
 
+if(ALICEVISION_BUILD_SFM)
+    find_package(E57Format)
```

Comment:
```suggestion
    find_package(E57Format CONFIG)
```
as it provides the `<PackageName>Config.cmake` file

Reply:
Also remember to add the build option in the build dependencies script

Reply:
Done.
The dependencies will be updated in another dedicated PR; for now this one will only add the optional dependency to libe57format.